examples/Viewer.tsx

```diff
@@ -146,15 +140,10 @@ const initialState: ViewerState = {
     selectionStateInfo: {
         highlightedAgents: [],
         hiddenAgents: [],
-        colorChangeAgents: [],
+        colorChanges: { agents: [], color: "" },
```

Comment:
I thought it was better to keep these data fields together since it's all connected rater than have the new color separated out